internal/log/level.go

```diff
@@ -36,17 +36,17 @@ const (
 func (level Level) String() (s string) {
 	switch level {
 	case Trace:
-		return "TRCE"
+		return "TRACE"
 	case Debug:
-		return "DBUG"
+		return "DEBUG"
 	case Info:
-		return "INFO"
+		return "INFO "
 	case Warn:
-		return "WARN"
+		return "WARN "
 	case Error:
-		return "EROR"
+		return "ERROR"
 	case Critical:
-		return "CRIT"
+		return "CRIT "
```

Comment:
Shouldn't we change this to `CRITICAL`?
Opinions maybe @kishansagathiya @danforbes 🤔 
But then we'll need 4 spaces suffix for all other levels (which is fine but still). Maybe @jimjbrettj you can check how the code concat the level with the next string, it might be `\t` and in that case just change it to a single space? That would thus look like `4`+`1` spaces in most log lines, and to `1` space only for `CRITICAL` level.

Reply:
I would probably prefer `CRITICAL` but it's not a hill I'd die on.

Reply:
@qdm12 updated!